installer/conf/omsagent.d/npmd.conf

```diff
@@ -1,9 +1,9 @@
 <source>
     type npmd
     tag oms.npmd
-    location_unix_endpoint  "/var/opt/microsoft/omsagent/run/npmdagent.sock"
-    location_control_data   "/etc/opt/microsoft/omsagent/conf/npmd_agent_config.xml"
+    omsadmin_conf_path      "%CONF_WS_DIR%/omsadmin.conf"
```

Comment:
Wrong string, it should be `%CONF_DIR_WS%;` see [monitor.conf](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/installer/conf/omsagent.d/monitor.conf#L22) for an example